test_unstructured/documents/test_elements.py

```diff
@@ -94,16 +94,8 @@ def test_convert_coordinates_to_new_system(
     assert element.metadata.coordinates.system == coord2
 
 
-@pytest.mark.parametrize(
-    ("coordinates", "coordinate_system"),
-    [
-        (None, None),
-        (((1, 2), (1, 4), (3, 4), (3, 2)), None),
-        (None, RelativeCoordinateSystem()),
```

Comment:
The tested method returns None if `self.metadata.coordinates` is None:
```
def convert_coordinates_to_new_system(
        self,
        new_system: CoordinateSystem,
        in_place=True,
    )
```

Parameterizing this test with the second and third pairs of arguments would throw a ValueError when trying to construct the element. This expectation is verified in `test_element_constructor_coordinates_points_absent` and `test_element_constructor_coordinates_system_absent`